pkg/ovs/openflow/ofctrl_group.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	"fmt"
+
+	"github.com/contiv/libOpenflow/openflow13"
+
+	"github.com/contiv/ofnet/ofctrl"
+)
+
+type ofGroup struct {
+	ofctrl  *ofctrl.Group
+	bridge  *OFBridge
+	builder *ofGroup
+}
+
+func (g *ofGroup) Reset() {
+	g.ofctrl.Switch = g.bridge.ofSwitch
+}
+
+func (g *ofGroup) Add() error {
+	return g.ofctrl.Install()
+}
+
+func (g *ofGroup) Modify() error {
+	return g.ofctrl.Install()
+}
+
+func (g *ofGroup) Delete() error {
+	return g.ofctrl.Delete()
+}
+
+func (g *ofGroup) Type() EntryType {
+	return GroupEntry
+}
+
+func (g *ofGroup) KeyString() string {
+	return fmt.Sprintf("group_id:%d,bucket_num:%d", g.ofctrl.ID, len(g.ofctrl.Buckets))
```

Comment:
I don't think it is a good idea to add the bucket count in the key string. Otherwise every time adding/deleting a bucket, the KeyString is changed, and the cache(if exists) will add new ones but not override the existing one.

Reply:
Make sense, IMO, group ID is enough. Please figure the issue out if you have further concerns about the change.